apps/web/src/shared/ui/sheet-detail-table.ui.tsx

```
@@ -91,50 +94,55 @@ type RenderFieldMap<T extends SheetDetailTableRow> = Record<
+                  (option) => option.key === bKey,
+                ),
+            )
+            .map((key) => {
+              const option = (row as SelectableRow).options.find(
+                (option) => option.key === key,
+              );
+              return (
+                <Badge
+                  variant={option?.color ? 'bold' : 'subtle'}
+                  className={option?.color ? BADGE_COLOR_MAP[option.color] : ''}
+                >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)